concepticondata/conceptlists/Vejdemo-2016-167.tsv

```diff
@@ -0,0 +1,168 @@
+Vejdemo-2016-167-56	MOUNTAIN	639	56	mountain	2.75	0.918347045	0.752286585	0.697247706	1.356877323	1.058355438	0.726968174	82		4.96	7772	4.12	2	6.15	Noun
+Vejdemo-2016-167-57	MOUTH	674	57	mouth	2.78	1.068404846	0.940358232	0.887406172	1.519702602	1.00795756	0.986599665	88		4.74	6672	4.14	8	3.58	Noun
+Vejdemo-2016-167-58	NAME	1405	58	name	0.47	1.072468151	1.074695122	1.964970809	1.194052045	0.453580902	0.675041876	292	4.2	3.5	5712	3.04	6	3.68	Noun
+Vejdemo-2016-167-59	NARROW	1267	59	narrow	2.89	1.249381441	1.128429878	0.380316931	1.356877323	1.51193634	1.869346734	73		3.04	7044	4.53	5	7.57	Adjective
+Vejdemo-2016-167-60	NECK	1333	60	neck	3.57	0.420290042		0.380316931	0.379925651	0.453580902	0.467336683	63	6.5	5	7.29	3.65	5	3	Noun
+Vejdemo-2016-167-61	NEW	1231	61	new	0.6	1.341701485	1.450838415	0.824020017	1.519702602	1.51193634	1.40201005	686	3	2.81	6051	5.14	10	4.72	Adjective
+Vejdemo-2016-167-62	NIGHT	1233	62	night	0.76	0.231974547	0.161204268	0.253544621		0.201591512	0.311557789	359	5.8	4.52	6.2685	3.57	8	3.61	Noun
+Vejdemo-2016-167-63	NOSE	1221	63	nose	1.49	1.060786693	0.725419207	1.838198499	0.434200743	1.00795756	1.298157454	78	6.4	4.89	7242	3.1	8	2.95	Noun
+Vejdemo-2016-167-64	OLD	1229	64	old	2.53	1.712510925	2.229992378	2.852376981	1.085501859	1.51193634	0.882747069	422	4.3	2.72	6.9555	4.48	8	3.72	Adjective
+Vejdemo-2016-167-65	PERSON	683	65	person	4.81	0.955566891	0.483612805	1.394495413		0.957559682	0.986599665	1086		4.72	5.7775	3.71	3	4.67	Noun
+Vejdemo-2016-167-66	RED	156	66	red	2.26							152	6	4.24	7738	5.02	3	3.68	Adjective
+Vejdemo-2016-167-67	RIGHT	1019	67	right	2.61							429		3.47	5448	5	3	4.346085	Adjective
+Vejdemo-2016-167-68	CORRECT (RIGHT)	1725	68	right correct	3.91	1.440721559	1.585175305	0.380316931	1.899628253	2.923076923	0.415410385	94	3.6	3.47	5448	5	12		Adjective
+Vejdemo-2016-167-69	RIVER	666	69	river stream brook	3.69	0.614277532		0.380316931	0.488475836	0.705570292	0.882747069	115		4.89	8.6355	4.22	1		Noun
+Vejdemo-2016-167-70	ROAD	667	70	road	4.86	1.511196739	0.349275915	3.169307756	1.573977695	0.957559682	1.505862647	318	6.6	4.75	8.3685	3.81	2	4.55	Noun
+Vejdemo-2016-167-71	ROOT	670	71	root	1.75	0.629759917	0.564214939	1.077564637	0.379925651	0.50397878	0.623115578	44	6	4.34	5681	3.62	8	5.94	Noun
+Vejdemo-2016-167-72	ROPE	1218	72	rope	6.09	0.487999433	0.188071646	0.316930776	0.705576208	0.554376658	0.675041876	29	6.6	4.93	4.8825	4.58	2	5.44	Noun
+Vejdemo-2016-167-73	ROTTEN	1728	73	rotten	4.76	1.421712038	2.176257622	0.760633862	1.465427509	1.51193634	1.194304858	8		3.87	4004	4.9	3	6.95	Adjective
+Vejdemo-2016-167-74	SALT	1274	74	salt	1.01	0.260085729	0.376143293	0.190158465	0.271375465	0.151193634	0.311557789	25	6.2	4.89	7.9325	4.53	3	5.05	Noun
+Vejdemo-2016-167-75	SAND	671	75	sand	4.26		0		0	0	0	41	6.7	5	8.0265	3.43	2	4.63	Noun
+Vejdemo-2016-167-76	SEA	1474	76	sea ocean	2.53	0.805628083	0.537347561	0.824020017	0.92267658	0.705570292	1.038525963	127	6.4	4.79	8272	2.8	3		Noun
```

Comment:
we have sea or ocean: http://concepticon.clld.org/parameters/2282 2282, SEA OR OCEAN.

Reply:
Shall I then map it to the SEA OR OCEAN (2282) concept set? I took [the Swadesh list (1952)](https://concepticon.clld.org/contributions/Swadesh-1952-200) as a reference, where this same concept is mapped to [the SEA concept (1474)](https://concepticon.clld.org/parameters/1474). In contrast, the SEA OR OCEAN concept set is mapped to just 3 concepts and, for this reason, I thought it might be somewhat inconsistent since the Vejdemo list is almost entirely based on the 200-item Swadesh's list.

Reply:
@LinguList since we've figured out that the Vejdemo list is built upon the list by [Pagel et al. (2007)](https://concepticon.clld.org/contributions/Pagel-2007-200), would it still make sense to map the concept sea_ocean to the concept set [SEA OR OCEAN (2282)](https://concepticon.clld.org/parameters/2282) given that the exact same concept in Pagel's list is mapped to [SEA (1474)](https://concepticon.clld.org/parameters/1474)?

Reply:
You're right, you could in fact just follow all mappings we used for pagel 2007!